src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -4408,9 +4427,15 @@ private void AddSynthesizedTupleMembersIfNecessary(MembersAndInitializersBuilder
                                                                                       initializer,
                                                                                       this,
                                                                                       DeclarationModifiers.Private | (property.IsStatic ? DeclarationModifiers.Static : 0),
-                                                                                      backingField);
+                                                                                      backingField); // PROTOTYPE(semi-auto-props): Is field-keyword BackingField needed here?
```

Comment:
>// PROTOTYPE(semi-auto-props): Is field-keyword BackingField needed here? [](http://example.com/codeflow?start=101&length=73)

This prototype comment is probably going to be addressed by the changes suggested above around the ```backingField``` local.

Reply:
> > // PROTOTYPE(semi-auto-props): Is field-keyword BackingField needed here?
> 
> This prototype comment is probably going to be addressed by the changes suggested above around the `backingField` local.

Not really. My purpose here is to know what exactly this is used for and add tests.

Reply:
> Not really. My purpose here is to know what exactly this is used for and add tests.

Sounds good. Thanks for the clarification.

Reply:
Consider making PROTOTYPE comments informative enough so that another person could address it. Be able to understand the concern, etc.